### ISSUE TITLE
Подсветка текущего документа в оглавлении

### ISSUE THREAD

**Issue by alyuev** [tigran123/urantia-book-explorer#17]
Добавить в оглавлении подсветку строки с выбранным документом, когда по нему кликнули для просмотра.

**Comment by tigran123** [comment]
Только строки с документом? А я думал раскрыть до самого глубокого уровня (секции) и скролить эту секцию к верхушке столбца Оглавление.

**Comment by alyuev** [comment]
Да, но ведь открывается именно документ, целиком. Далее идет просто позиционирование внутри документа.

**Comment by tigran123** [comment]
Почему документ целиком? Из столбца Результаты Поиска можно открыть любой индивидуальный параграф в конкретной секции конкретного документа. Разве не логичнее (хотя и сложнее) сначала раскрыть дерево оглавления до уровня данной секции (главы) и потом её как-то выделить?

**Comment by alyuev** [comment]
При переходе из результатов всё равно открывается весь документ, которому принадлежит данный параграф. Поэтому не вижу особого смысла выделять в меню текущую главу/секцию.

**Comment by tigran123** [comment]
Перейти можно только к конкретному параграфу. Да, конечно, весь документ открывается в соответствующем текстовом столбце. Но, ты можешь захотеть перейти к соседней главе в том же документе. Вместо того, чтобы скролить текст и смотреть "какие у нас тут ещё главы есть", ты сможешь посмотреть влево и сразу увидеть список этих глав.

**Comment by alyuev** [comment]
Да, согласен, но это сложнее в реализации. Нужно ж еще динамически отслеживать текущую секцию, чтобы ее подсветить.

**Comment by tigran123** [comment]
Ты имеешь в виду во время ручного скроллинга? Я думал его проигнорировать и показывать только в случае прямого клика откуда-либо.

**Comment by alyuev** [comment]
А... Ну так да, можно.... Без отслеживания скроллинга.

**Comment by tigran123** [comment]
Кстати, насчёт отслеживания скроллинга --- я даже думал о функции "полная синхронизация столбцов", т.е. комбинирование его с твоим предложением насчёт кликов из Оглавления/Результатов Поиска, чтобы все тексты прыгали туда.
